dags/multipod/maxtext_convergence.py

```diff
@@ -46,21 +46,31 @@
 
   base_convergence_command = f"bash end_to_end/test_convergence_1b_params.sh OUTPUT_PATH={base_output_directory} DATASET_PATH={dataset_path} LOSS_THRESHOLD={loss_threshold} STEPS={steps}"
   convergence_tests = {
-      "maxtext-convergence-bf16": ((base_convergence_command),),
-      "maxtext-convergence-int8": (
+      "maxtext-converge-bf16": ((base_convergence_command),),
+      "maxtext-converge-int8": (
           (f"export M_QUANTIZATION=int8; {base_convergence_command}"),
       ),
   }
 
   for test_name, run_command in convergence_tests.items():
-    maxtext_v4_configs_test = gke_config.get_gke_config(
-        tpu_version=TpuVersion.V4,
-        tpu_cores=128,
-        tpu_zone=Zone.US_CENTRAL2_B.value,
-        cluster_name=ClusterName.V4_128_MULTISLICE_CLUSTER.value,
-        time_out_in_min=300,
-        test_name=test_name,
-        run_model_cmds=run_command,
-        docker_image=DockerImage.MAXTEXT_JAX_STABLE.value,
-        test_owner=test_owner.MATT_D,
-    ).run()
+    maxtext_v4_configs_test = (
+        maxtext_sweep_gke_config.get_maxtext_sweep_gke_config(
```

Comment:
Just curious why we need the sweep?

Reply:
The sweep isn't needed here, just reusing the metric_config set up. Would it be better to modify `get_gke_config` instead so there's less confusion?

Reply:
Ahhh I see... Yeah, we can add it into `get_gke_config` if it's easy, but we should keep it as an optional config.

One small callout - `base_output_directory` is generated at dag parse time, and now that we share it between tasks it could change during DAG execution. Not a huge issue because this test runs early in the morning UTC and the format is using just the date, but this is something we'll need to address later with generated GCS paths.

Reply:
Done! Added the `metric_config` in `get_gke_config` only when the base_output_directory and aggregation_strategy are specified.

Agreed, that is a good callout. We'll definitely need a more unified and robust way for generating values that contain dates and time